Microsoft.Toolkit.Uwp.UI.Controls/GridSplitter/GridSplitter.xaml

```diff
@@ -23,6 +23,7 @@
         <Setter Property="MinHeight" Value="16"></Setter>
         <Setter Property="Background" Value="{ThemeResource SystemControlHighlightChromeHighBrush}"></Setter>
         <Setter Property="GripperForeground" Value="{ThemeResource SystemControlForegroundAltHighBrush}" />
+        <Setter Property="AutomationProperties.Name" Value="GridSplitter" />
```

Comment:
We should have this string in the resource file as well.  I'm wondering if we need a MarkupExtension to provide this value (which would just call the GetLocalized method I provided with the parameter to the extension)...  We'd have to try it though.

Reply:
Since you set this in the constructor, you don't need this here anymore.